### PR TITLE
removing json strings

### PR DESCRIPTION
**Pull request by unixslayer**
resolves prooph/event-store#401

### CODE REVIEW

File: tests/when_enabling_projections.php [hunk 1]
```diff
@@ -60,7 +60,7 @@ protected function when(): Generator
     }
 
     /** @test */
-    public function should_stop_the_projection(): Generator
+    public function should_run_the_projection(): Generator
```
Comment (prolic) [inline tests/when_enabling_projections.php:63]: good catch

File: src/Projections/ProjectionsClient.php [hunk 2]
```diff
@@ -415,16 +448,44 @@ public function getPartitionState(
...
+
+        $promise->onResolve(function (?Throwable $exception, ?string $body) use ($deferred): void {
+            if ($exception) {
+                $deferred->fail($exception);
+
+                return;
+            }
+
+            if (null === $body) {
+                $deferred->resolve(new UnexpectedValueException('No content received'));
```
Comment (prolic) [inline src/Projections/ProjectionsClient.php:460]: deferred->fail

File: src/Projections/ProjectionsClient.php [hunk 3]
```diff
@@ -434,17 +495,45 @@ public function getResult(
...
```
Comment (prolic) [inline src/Projections/ProjectionsClient.php:507]: fail again

File: src/Projections/ProjectionsClient.php [hunk 4]
```diff
@@ -455,16 +544,44 @@ public function getPartitionResult(
...
```
Comment (prolic) [inline src/Projections/ProjectionsClient.php:556]: fail

File: src/Projections/ProjectionsClient.php [hunk 5]
```diff
@@ -474,16 +591,45 @@ public function getStatistics(
...
```
Comment (prolic) [inline src/Projections/ProjectionsClient.php:603]: ...

File: src/Projections/ProjectionsClient.php [hunk 6]
```diff
@@ -493,6 +639,24 @@ public function getQuery(
...
```
Comment (prolic) [inline src/Projections/ProjectionsClient.php:651]: ...

**Comment by enumag** [comment]
@prolic you could have used suggestions for that ;-)

**Comment by prolic** [comment]
I fixed it myself via 157d76302ff45b7769efa393636c9bb48cfb4716

Thanks @unixslayer

**Pull request merged**